docs/src/learning_rate_scheduler.md

```diff
@@ -0,0 +1,69 @@
+
+![Solution ensembles](assets/ensemble_compare_scheduler.png)
+
+*Left: The true model over ``[0,2\pi]`` (black), and the ensembles from different schemes (colors).
+Right: The noisy observation (black), the distribution it was sampled from (gray-ribbon), and the corresponding ensemble-mean approximation given from each scheduler (colors).*
+
+To assess the timestepping we show the convergence plot against the algorithm iteration we measure two quantities.
+- error (solid) is defined by ``\frac{1}{N_{ens}}\sum^{N_{ens}}_{i=1} \| \theta_i - \theta^* \|^2`` where ``\theta_i`` are ensemble members and ``\theta^*`` is the true value used to create the observed data.
+- spread (dashed) is defined by ``\frac{1}{N_{ens}}\sum^{N_{ens}}_{i=1} \| \theta_i - \bar{\theta} \|^2`` where ``\theta_i`` are ensemble members and ``\bar{\theta}`` is the mean over these members.
+
+![Error vs spread](assets/error_vs_spread_over_iteration_compare_scheduler.png)
+
+*Left: the error and spread of the different timesteppers at over iterations of the algorithm for a single run.
+Right: the error and spread of the different timesteppers at their final iterations, averaged from 100 different draws of initial conditions.*
+
+From this plot there are several take-home messages.
+ - Termination at ``T=1``. DMC with termination (purple), closely mimics a small-fixed timestep (green) that finishes stepping at ``T=1``. Both retain more spread than other approaches, and DMC is far more efficient, typically terminating after around 10-20 steps, where fixed-stepping takes 50. We see that (for this experiment) this is a conservative estimate, as continuing to solve (e.g. brown) until later times often leads to a better error while still retaining similar "error vs spread" curves (before inevitable collapse).
+ - Large fixed step (orange). This is very efficient, but can get stuck in local minima when too large, it collapses the ensemble. Note that even on average it gives lower error to the true parameters than DMC. 
+ - Continuation ``T \to \infty``. Both EKSStable and DMC with continuation schedulers, perform very similarly. Both retain good ensemble spread for during convergence, and collapse after finding a local optimum. This optimum on average has the best error to the true parameters in this experiment. They appear to consistently find the same optimum as ``T\to\infty`` but DMC finds this in fewer iterations.
+ - Adaptivity. The fixed-step may eventually find the same optimum as EKS/DMC, but after 50 iterations ``T\sim 10^{12} - 10^{15}`` in the adaptive algorithms, while the fixed stepper has reached ``T=25``. On the other end of the scale, the initial timesteps selected are often on the order of ``\Delta t \sim 10^{-8} - 10^{-6}``. The implication is that adaptivity is likely very useful for efficiency of these algorithms.
```

Comment:
I am not sure I understand how the efficiency follows from the discrepancy in times.

Reply:
The adaptive methods reached their minimum value at `N_{it} = 20` or so, this  corresponded to a  very large algorithm time `T>>1`. It appears likely that the fixed timestepper will reach this same minimum but will take in `N_{it} >> 20` as it cannot take larger steps. So they are very efficient.

I can rewrite this to explain it better